a2/a2/main.py

```diff
@@ -74,7 +74,7 @@ def get_goals(width, height):
 
 if __name__ == "__main__":
     parser = argparse.ArgumentParser(prog="ᵡ-puzzle solver", description="A program to find the solution of a ᵡ-puzzle using a chosen algorithm")
-    parser.add_argument("--inputpuzzle", "-in", metavar="input puzzle", help="the path to the file containing the puzzles to solve", type=open, default="in/puzzle.txt")
+    parser.add_argument("--inputpuzzle", "-in", metavar="input puzzle", help="the path to the file containing the puzzles to solve", type=open, default="in/puzzle_short.txt")
```

Comment:
I think we should keep it at puzzle.txt and change the file names instead

Reply:
it was already puzzle_short when I started. I changed it when testing my stuff then changed it back to what you had it before pushing